### PR TITLE
Resolve build bugs and instructions to connect to server(From mobile devices).

### PR DESCRIPTION
**Pull request by ganeshpatro321**
This PR solves server build, and adds instructions to connect to server through local tunnels.

### CODE REVIEW

**Comment by ganeshpatro321** [comment]
@aquibbaig Review.

**Comment by aquibbaig** [comment]
LGTM!

File: lib/utils/websocket.dart [hunk 1]
```diff
@@ -38,7 +38,7 @@ class WebSocket {
     // Initiate communication 
     // To Connect to the Localhost in the App, Read this following
     // https://stackoverflow.com/questions/4779963/how-can-i-access-my-localhost-from-my-android-device
-    _channel = IO.io('http://localhost:3000/', <String, dynamic>{
+    _channel = IO.io('http://56d6501f.ngrok.io/', <String, dynamic>{
```
Comment (aquibbaig) [inline lib/utils/websocket.dart]: @ganeshpatro321 How does this work actually?

Comment (ganeshpatro321) [reply lib/utils/websocket.dart]: Read the readme. I have described steps to create a local tunnel. It basically exposes your local server through a tunnel to the internet, and you can call your apis through that link. @aquibbaig

Comment (aquibbaig) [reply lib/utils/websocket.dart]: Thanks @ganeshpatro321

**Pull request merged**